src/client/src/apps/MainRoute/store/combineReducers.ts

```diff
@@ -1,15 +1,18 @@
 import { combineReducers } from 'redux'
-
+import { platform } from 'rt-platforms'
 import { reducers } from 'apps/MainRoute/data'
 import { analyticsReducer } from '../widgets/analytics/index'
 import { blotterReducer } from '../widgets/blotter/index'
 import { spotTileDataReducer } from '../widgets/spotTile/index'
 
+const customReducers = platform.customReducers
+
 const rootReducer = combineReducers({
   ...reducers,
+  ...customReducers,
```

Comment:
not sure if I'm onboard with prefixing things with custom when the rest are semantically named